python/llm/src/bigdl/llm/transformers/speculative.py

```diff
@@ -231,6 +231,10 @@ def speculative_generate(self,
 
     _enable_ipex = os.getenv("BIGDL_OPT_IPEX")
     _enable_ipex = (_enable_ipex is not None) and (_enable_ipex.lower() == "true")
+    if _enable_ipex:
+        if not (self.config.model_type == 'baichuan' or 'llama' in self.config.model_type):
+            invalidInputError(False, "BigDL Speculative Decoding with IPEX BF16 only supports \
+                                      Llama and Baichuan2-13b models currently.")
```

Comment:
how about baichuan1 and baichuan2-7b?

Reply:
Currently baichuan1 and baichuan2-7b are not supported yet.